bin/live/pycbc_live_supervise_collated_triggers

```diff
@@ -0,0 +1,432 @@
+    "fit-over-combined-days",
+    "plot-fits-format",
+    "variable-fit-over-param",
+    "variable-dq-file",
+    'collated-triggers-format',
+    'top-directory',
+]
+
+# These options are for the collection of triggers into a merged file
+trigger_collation_options = [
+    'bank-file',
+    'date-directories',
+    'date-directory-format',
+    'file-identifier',
+    'ifos',
+    'template-cuts',
+    'trigger-cuts',
+    'verbose',
+]
+
+# These options are for the fit_by_template code
+fit_by_template_options = [
+    'approximant',
+    'bank-file',
+    'f-lower',
+    'fit-function',
+    'gating-veto-windows',
+    'log-prune-param',
+    'min-duration',
+    'prune-param',
+    'prune-bins',
+    'prune-number',
+    'sngl-ranking',
+    'stat-threshold',
+    'statistic-files',
+    'statistic-keywords',
+    'verbose',
+]
+
+# These options are for the fit_over_multiparam code
+fit_over_multiparam_options = [
+    'approximant',
+    'bank-file',
+    'f-lower',
+    'fit-param',
+    'log-param',
+    'min-duration',
+    'output-fits-by-template',
+    'smoothing-method',
+    'smoothing-keywords',
+    'smoothing-width',
+    'verbose',
+]
+
+# These options are for the plot_template_bank_corner code
+plot_bank_corner_options = [
+    'bank-file',
+    'color-parameter',
+    'dpi',
+    'maxs',
+    'mins',
+    'mpl-style',
+    'parameters',
+    'plot-histogram',
+    'verbose',
+]
+
+# These options are for the bin_trigger_rates_dq code
+#bin_dq_options = [
+#    'template-bins-file',
+#    'stat-threshold',
+#    'sngl-ranking',
+#    'verbose',
+#]
+
+# These options are for the plot_dq_flag_likelihood
+#plot_dq_options = [
+#    'dq-file',
+#    'dq-label',
+#    'verbose',
+#]
+
+all_options = control_options + trigger_collation_options \
+    + fit_by_template_options + fit_over_multiparam_options \
+    + plot_bank_corner_options #+ bin_dq_options \
+#    + plot_dq_options
+
+all_options = set(all_options)
+
+def read_options(args):
+    """
+    read the options into a dictionary
+    """
+    option_values = {opt: None for opt in all_options}
+
+    with open(args.config_file, 'r') as conf_file:
+        all_lines = conf_file.readlines()
+
+    for line in all_lines:
+        # Ignore whitespace and comments
+        line = line.strip()
+        if not line:
+            continue
+        if line.startswith(';'):
+            continue
+
+        option, value = line.split('=')
+
+        option = option.strip()
+        if option not in all_options:
+            logging.warning("Option %s unrecognised, ignoring", option)
+            continue
+
+        if value.strip() == '':
+            # Option is a flag - Store as True
+            value = True
+        elif len(value.split()) > 1:
+            # Option is a list
+            value = value.split()
+        else:
+            # Option is a single value
+            value = value.strip()
+
+        option_values[option] = value
+
+    return option_values
```

Comment:
Can we rely on configparser for this functionality?

Reply:
If not, can we at least reduce the duplication of the same code in `pycbc_live_supervise_single_significance_fits`?